.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "0.12"
-  - "0.11"
-  - "0.10"
+  - "node"
+  - "6"
+  - "4"
```

Comment:
We already have an editorconfig in every repo I can see. I'm not sure why it didn't work here. I'm using vs code, and a quick test makes it look like this should've been added fine. Think I might've edited in vim and I might not have a plugin working there?

I've fixed this file anyway, I'll fix my vim, and then ignore this unless it comes up again.